README.md

```diff
@@ -17,7 +17,10 @@ Currently, these metric sources are implemented:
 
 | Old Percona MongoDB exporter                                                  |
 |:------------------------------------------------------------------------------|
-| old 0.1x.y version (ex `master` branch) is moved to the `release-0.1x` branch |
+| old 0.1x.y version (ex `master` branch) is moved to the `release-0.1x` branch.|
+| If you considering migrating from the old version of the exporter  - you can  |
+| use flag `--compatible-mode` to expose metrics in the old format. This will   |
```

Comment:
The `format` is the same, the prometheus metric format. 
The `--compatible-mode` exposes new metrics using the new and old metric names at the same time, which means that any script or Grafana dashboard using a specific metric name from the 1.x version will still work.